Sources/Logging/Logging.swift

```diff
@@ -1107,21 +1159,25 @@ public struct StreamLogHandler: LogHandler {
     internal typealias _SendableTextOutputStream = TextOutputStream & Sendable
 
     /// Factory that makes a `StreamLogHandler` to directs its output to `stdout`
+    @Sendable
```

Comment:
Why is this necessary?

Reply:
Because it's used as a parameter to LoggingSystem.bootstrap which requires it's parameter to be sendable.

Reply:
This is a static function, its parameter _is_ `Sendable`.

Reply:
Ok - this is now removed using the closure trick we discussed offline.